superset-frontend/src/views/CRUD/data/database/DatabaseModal/DatabaseConnectionForm.tsx

```diff
@@ -107,52 +185,30 @@ const DatabaseConnectionForm = ({
         Need help? Learn more about connecting to {name}.
       </p>
     </StyledFormHeader>
-    <div css={formScrollableStyles}>
+    <div
+      // @ts-ignore
+      css={(theme: SupersetTheme) => [
```

Comment:
we have to look into why this isn't using `InterpolationWithTheme` from @emotion/core and picking up `Interpolation` from @emotion/react instead. It seems to only happen this way with regular html elements. AntD components use InterpolationWithTheme.